servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -999,7 +1109,7 @@ void executePutPolicy(ResourceContext ctx, String domainName, String policyName,
 
                 // retrieve our original policy
 
-                Policy originalPolicy = getPolicy(con, domainName, policyName);
+                Policy originalPolicy = getPolicy(con, domainName, policyName, null);
```

Comment:
The function executePutPolicy is only called from endpoints that use the active version.
I prefer not to use the version specified by the user because if he used these endpoints he meant to use the active version.